docs/en/pact-functions.md

```diff
@@ -1634,13 +1634,24 @@ pact> (env-hash (hash "hello"))
 *keys*&nbsp;`[string]` *&rarr;*&nbsp;`string`
 
 
-Set transaction signature KEYS.
+Set transaction signature KEYS. See 'env-sigs' for setting keys with associated capabilities.
 ```lisp
 pact> (env-keys ["my-key" "admin-key"])
 "Setting transaction keys"
 ```
 
 
+### env-sigs {#env-sigs}
+
+*sigs*&nbsp;`[object:*]` *&rarr;*&nbsp;`string`
+
+
+Set transaction signature keys and capabilities. SIGS is a list of objects with "key" specifying the sig key, and "clist" specifying a list of capabilities.
+```lisp
+(env-sigs [{'key: "my-key", 'clist: [(accounts.USER_GUARD "my-account")]}, {'key: "admin-key", 'clist: []}
```

Comment:
So are `env-keys` and `env-sigs` equivalent, except that `env-sigs` lets you specify a capability associated with a signature? Why are they different functions?

Reply:
Just for backward compat, and because `env-keys` is a bad name and it gets even more confusing when you start adding the caps. 

I could also overload `env-keys` or even make it a synonym, `env-sigs` is actually clearer in it's own way. What do you think? @emilypi ?